CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
   - Avoid basing results on old analysis.
   - Add new setting `:completion :analysis-type` to choose between `:fast-but-stale`(default) or `:slow-but-accurate`, this should define whether completion should wait for changes that may still happening, this by default reverts the behavior introduced after #1425. #1487
 
+- API/CLI
```

Comment:
That's actually related to `Editor`, API/CLI is more related to `clojure-lsp.api` features like the CLI

Reply:
Updated.